### PR TITLE
fix: remove deprecated sf env commands

### PR DESCRIPTION
**Pull request by mshanemc**
remove some busted references to sf env commands that are now in `org` topic

### CODE REVIEW

File: src/hubAuth.ts [hunk 1]
```diff
@@ -81,13 +81,13 @@ export const testkitHubAuth = (homeDir: string, authStrategy: DevhubAuthStrategy
     const jwtKey = prepareForJwt(homeDir);
 
     const results = shell.exec(
-      `sf login jwt org --set-default-dev-hub --username ${env.getString(
+      `sf org login jwt --set-default-dev-hub --username ${env.getString(
```
Comment (peternhale) [inline src/hubAuth.ts:84]: command with `:` for consistency (and command lookup perf)

File: src/testSession.ts [hunk 2]
```diff
@@ -192,7 +192,7 @@ export class TestSession<T extends TestSessionOptions = TestSessionOptions> exte
       const usernameOrAlias = configResults.result.find((org) => org.name === 'target-dev-hub')?.value;
       if (usernameOrAlias) {
         const displayEnv = shell.exec(
-          `sf env display -e ${usernameOrAlias} --json`,
+          `sf org display -o ${usernameOrAlias} --json`,
```
Comment (peternhale) [inline src/testSession.ts:195]: ditto `:`

File: src/testSession.ts [hunk 3]
```diff
@@ -284,7 +284,7 @@ export class TestSession<T extends TestSessionOptions = TestSessionOptions> exte
         if (this.orgsAliases.includes(org)) continue;
 
         this.debug(`Deleting test org: ${org}`);
-        const rv = shell.exec(`sf env delete scratch -o ${org} -p`, this.shelljsExecOptions) as shell.ShellString;
+        const rv = shell.exec(`sf org delete scratch -o ${org} -p`, this.shelljsExecOptions) as shell.ShellString;
```
Comment (peternhale) [inline src/testSession.ts:287]: and here

File: test/unit/testSession.test.ts [hunk 4]
```diff
@@ -340,7 +340,7 @@ describe('TestSession', () => {
       await session.clean();
 
       expect(restoreSpy.called).to.equal(true);
-      expect(execStub.firstCall.args[0]).to.equal(`sf env delete scratch -o ${username} -p`);
+      expect(execStub.firstCall.args[0]).to.equal(`sf org delete scratch -o ${username} -p`);
```
Comment (peternhale) [inline test/unit/testSession.test.ts:343]: and here

**Review by peternhale — CHANGES_REQUESTED** [review]

**Review by WillieRuemmele — APPROVED** [review]

**Pull request merged**